debugger.md

```diff
@@ -1,60 +1,48 @@
-﻿#Instructions for setting up the .NET Core debugger - PREVIEW
-*Updated on  March 1st 2016 - setup works for Linux & Mac.*
-
-***Currently working within Microsoft Corp-Net only.***
-
+﻿#Instructions for setting up the .NET Core debugger - EXPERIMENTAL PREVIEW
 Thank you for taking the time to try out our bits. 
 This page gives you detailed instructions on how to get your system running. 
-It will be updated on a regulary basis to always reflect the latest behaviour. Now let's get started.​
 
 ####Your Feedback​
-
 Please place your feedback [here](https://github.com/OmniSharp/omnisharp-vscode/issues). 
 
 ####Prerequisites / Known Issues / Notes
 * X64 only
-* Basic Debugging Setup successfully validated on Ubuntu Linux and Mac
-* Currently NOT working on Windows
 * In case you get this error on F5: *"No task runner configured -  Tasks.json could not be found"*, see the 'once for each project' section below. 
 * If you don’t have mono installed you won't get IntelliSense. 
-* **Hint:** Setup for Debugging CLR on OSX changed fundamentally end of February 2016. For existing projects please make sure to update your launch.json file according to these steps.​
 
 ###First Time setup
 1. Get Visual Studio Code
-    * Install Visual Studio Code. Pick the latest insider version from here: https://code.visualstudio.com/insiders
-    * **Hint:** While we are in a pre-release phase vscode might ask you to install a C# extension from the marketplace. DO NOT INSTALL IT! This extension doesn't contain debugger bits yet and might bring your system in a weird state.   
+ * Install Visual Studio Code (VSC). Pick the latest VSC version from here: https://code.visualstudio.com/insiders Make sure it is at least 0.10.10.       
 2. Install Dotnet CLI
  * Install Dotnet CLI following the instructions here:  http://dotnet.github.io/getting-started  
  * **Hint for Mac**: Dotnet CLI requires openSSL to work. Don't forget this! Execute: `brew install openssl`
```

Comment:
On Windows we need to tell people to install something newer than what is on the web site.